### PR TITLE
Monthly payment worksheet: calculations

### PR DESCRIPTION
**Pull request by virginiacc**
Adds monthly payment calculations

## Additions

- Add monthly payment calculations file
- Add basic tests for calculations

## Testing

- Run `grunt test` to see results of initial tests for calculations module

## Review

- @cfarm 
- @mthibos 

## Notes
- The calculations currently call each other where necessary, so they don't depend on previous calculations having been run to produce the data that is passed in to them.

- This PR contains the basic logic for the calculations. Formatting values/ensuring numerical values may follow in future PR.

- @mthibos Do we want to round these calculations? Or keep decimals, if a user enters them?

## Checklist

* [X] Changes are limited to a single goal (no scope creep)
* [X] Code can be automatically merged (no conflicts)
* [X] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [X] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by stephanieosan** [comment]
Pinging @contolini because I bet Cat is already on the road.

**Comment by contolini** [comment]
I'll review this today.

On Tuesday, October 13, 2015, Stephanie Osan <notifications@github.com>
wrote:

> Pinging @contolini <https://github.com/contolini> because I bet Cat is
> already on the road.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/owning-a-home/pull/640#issuecomment-147815331>.
>


-- 
Chris

File: src/static/js/modules/monthly-payment-calc.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+monthly.estimatedTotalPayment = function (data) {
+  return monthly.availableHousingFunds(data) - parseFloat(data.condoHOA);
+}
+
+monthly.taxesAndInsurance = function (data) {
+  var annualTaxesAndInsurance = (parseFloat(data.homePrice) * parseFloat(data.propertyTax / 100))
+                              + parseFloat(data.homeownersInsurance);
+  var total = annualTaxesAndInsurance / 12;
+  return Math.floor(total) == total ? total : parseFloat(total.toFixed(2));
```
Comment (contolini) [inline src/static/js/modules/monthly-payment-calc.js]: What's the purpose of comparing `total` to its floor? To only return decimal notation if it's not an integer?

Comment (virginiacc) [inline src/static/js/modules/monthly-payment-calc.js]: @contolini Yes, that was the purpose. It should be replaced with a global approach to decimal place handling/rounding, though.

Comment (contolini) [inline src/static/js/modules/monthly-payment-calc.js]: Is the ternary necessary? Won't parseFloat preserve ints?

```js
var total = 500;
console.log(parseFloat(total.toFixed(2))); // returns 500
```

**Comment by contolini** [comment]
Aside from my minor nitpicking above, this looks :+1: to me.

**Comment by virginiacc** [comment]
@contolini Ok, I removed that line from the insurance calculation.

**Pull request merged**